tests/storage/snapshot.go

```diff
@@ -1335,27 +1326,11 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 				}
 			})
 
-			It("Bug #8435 - should create a snapshot successfully, currently fails as source remains locked", func() {
+			It("Bug #8435 - should create a snapshot successfully", func() {
```

Comment:
should remove the bug number? if this commit fixes it?

Reply:
Happy to if people don't think it's useful, my entire approach here is based on what [I previously did in the OpenStack Nova project](https://github.com/openstack/nova/commits/master/nova/tests/functional/regressions/test_bug_1937084.py) and might not be suitable for this project so I'm happy to change if required.